tests/launch.rs

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#![cfg(feature = "openssl")]
+
+use sev::certs::Chain;
+use sev::firmware::Firmware;
+use sev::launch::{HeaderFlags, Launcher, Policy};
+use sev::session::Session;
+
+use sev::Generation;
+
+use kvm_bindings::kvm_userspace_memory_region;
+use kvm_ioctls::{Kvm, VcpuExit};
+use mmarinus::{perms, Kind, Map};
+
+use std::convert::TryFrom;
+use std::fs::File;
+use std::io::Write;
+
+const CODE: &[u8] = &[
+    0xf4, // hlt
+];
+
+fn get_cert_chain(sev: &mut Firmware) -> Chain {
+    use codicon::Decoder;
+
+    // If you are running this test suite often without manipulating the platform
+    // state (i.e., platform reset) it is recommended that you export the entire
+    // chain to a file and use this environment variable to avoid getting rate-limited
+    // by the AMD web service (which will cause a panic).
```

Comment:
Maybe it would be better to just cache the certificate chain? We have `~/.cache` and `/var/cache` for that purpose. Caching would avoid having to bother the user with these details.